src/styles/main.scss

```diff
@@ -0,0 +1,58 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  max-width: 706px;
+  gap: 1px;
+
+  &__day {
+    position: relative;
+    display: none;
+    justify-content: center;
+    align-items: center;
+    width: $square-size;
+    height: $square-size;
+    background-color: #eee;
+    border: 1px solid;
+    transition-duration: 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+    }
+  }
+}
+
+@for $number from 1 through 31 {
+  .calendar :nth-child(#{$number})::before {
+    content: '#{$number}';
+    position: absolute;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+  }
+}
+
+@each $day-of-start, $margins in $day-start-styles {
+  .calendar--start-day-#{$day-of-start} :first-child {
+    margin-left: $margins;
+  }
+}
+
+@for $day-show from 29 to 32 {
+  .calendar--month-length-#{$day-show} :nth-child(-n + #{$day-show}) {
+    display: flex;
+  }
+}
```

Comment:
Why do you do it inside a loop?

Reply:
> Because I do display none for every day and this loop makes elements visible for quantity that sets in month-length modifier
> 
> <img alt="displays-in-calendar" width="1728" src="https://private-user-images.githubusercontent.com/172034539/359089497-2125c2e0-b597-46f4-a6f7-d943d5c0fb16.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwNzY4MDUsIm5iZiI6MTcyNDA3NjUwNSwicGF0aCI6Ii8xNzIwMzQ1MzkvMzU5MDg5NDk3LTIxMjVjMmUwLWI1OTctNDZmNC1hNmY3LWQ5NDNkNWMwZmIxNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxOVQxNDA4MjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lODZjYWU0NTIyZDczNzkzNjUzYTkxY2EzOWMzYjAyODQ5N2RjYTAzYzhlNWEwMGQ5Y2JkNzE4YzhkZWI4MGUwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Q7mMbW6NMw35BHJR_wmUHGP7rZITx2E9LpXcH9oOVOE">

Got it